lib/reservation.rb

```diff
@@ -0,0 +1,30 @@
+# #Reservation
+# A ticket stub with reservation details
+#check-in, check-out
+#As an administrator, I can reserve a room for a given date range
+require 'pry'
+require 'date'
+require 'awesome_print'
+require_relative 'room'
+
+class Reservation
+
+  attr_reader :start_date, :end_date, :room, :total_cost
+
+  def initialize(start_date, end_date, room)
+    @start_date = start_date
+    @end_date = end_date
+    @room = room
+    @total_cost = calculate_total_cost
+  end
+
```

Comment:
I would also suggest you add a method to tell if two reservations overlap, and another to tell if a reservation occurs on a specific date.